code/game/objects/effects/overlays.dm

```diff
@@ -135,7 +213,7 @@
 /obj/effect/overlay/temp/laser_target/cas
 	icon_state = "laser_target_coordinate"
 
-/obj/effect/overlay/temp/laser_target/cas/Initialize(mapload, effect_duration, named, assigned_squad = null)
+/obj/effect/overlay/temp/laser_target/cas/Initialize(mapload, named, assigned_squad = null)
```

Comment:
explain this

Reply:
I fucked up local. Same for wraith, should be fixed